packages/web/src/app/members/page.tsx

```diff
@@ -0,0 +1,71 @@
+interface MemberInfo {
+  rank: string;
+  benefits: string[];
+  requirements?: string[];
+}
+
+const membershipLevels: MemberInfo[] = [
+  {
+    rank: "Free Player",
+    benefits: [
+      "Access to basic game features",
+      "Play against other players",
+      "Basic orbs and powers",
+    ],
+  },
+  {
+    rank: "Premium Member",
+    benefits: [
+      "Access to all orbs and powers",
+      "Create private matches",
+      "Custom player profile",
+      "Ad-free experience",
+      "Priority server access",
+    ],
+    requirements: ["Monthly subscription", "Valid email verification"],
```

Comment:
I know that the original did state so, but I'm certain that this will cause confusion, so I would instead write something along the lines `Free registration on official Discord server`